Configurations/10-main.conf

```diff
@@ -311,9 +310,7 @@ sub vms_info {
     "solaris64-sparcv9-cc" => {
         inherit_from     => [ "solaris-sparcv7-cc", asm("sparcv9_asm") ],
         cflags           => add_before("-xarch=v9"),
-        lflags           => add_before("-xarch=v9"),
         bn_ops           => "BN_LLONG RC4_CHAR",
-        shared_ldflag    => "-xarch=v9 -G -dy -z text",
```

Comment:
So suggestion is to remove this, but not corresponding line in solaris64-x86_64-cc... Which one is right?

Reply:
Well, you already have `-xarch=v9` in cflags, and `-G -dy -z text` gets inherited from `solaris-sparcv7-cc`, so having them here is just redundant.  `solaris64-x86_64-cc` doesn't inherit its flags the same way, although it's true that `-xarch=generic64` is redundant there...  is that what you meant?

I didn't go in a all-out-clean-everything-meticulously, there were just a few things I happened to see that I cleaned out.  Are you asking me to be more thorough here?

Reply:
Ah! I see now... Keyword is that solaris64-x86_64-cc doesn't inherit. Thanks. As for -xarch=generic64 being redundant there. I didn't pay attention to *that*, but it *is* redundant. For consistency it could and probably should be removed.